src/components/TableCreator/common/utils.ts

```diff
@@ -1,27 +1,34 @@
 import { ColumnDef } from "@tanstack/react-table";
-import { BaseColumnConfig } from "./entities";
+import { ColumnConfig } from "../../../config/entities";
 
 /**
  * Builds a base column definition.
+ * The following properties are set by default:
+ * - Grouping is not enabled `enableGrouping: false`.
+ * - Sorting is enabled - `enableSorting: true`.
  * @param baseColumnConfig - Base column configuration.
  * @returns column definition.
  */
-export function buildBaseColumnDef<T, TValue>(
-  baseColumnConfig: BaseColumnConfig<T, TValue>
+export function buildBaseColumnDef<T>(
```

Comment:
Updating `buildBaseColumnDef` to be closer to using TanStack `ColumnDef` API... The goal will be to remove this function at some point!